views/LoginView.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  Text,
+  TextInput,
+  Keyboard,
+  View,
+  TouchableWithoutFeedback,
+  TouchableOpacity,
+} from 'react-native';
+import store from '../state/store/configureStore';
+import { useSelector } from 'react-redux';
+import Authentication from '../modules/Authentication';
+
+const LoginView = ({ navigation, route }) => {
+  const { errorMessage } = useSelector((state) => state);
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const auth = new Authentication({ host: '' });
+  const article = route.params.article;
+
+  const authenticate = async () => {
+    try {
+      await auth.signIn(email, password);
+      navigation.navigate('single article', { article: article });
+    } catch (error) {
+      console.log(error);
```

Comment:
what is the point of consolelog for user? user will never see it, I think you should delete it or display it on the page.

Reply:
IZ for all the hackers